<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QDialog" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">
text-align: center;
color: rgb(0, 0, 0);
font: 14pt &quot;Franklin Gothic&quot;;</string>
     </property>
     <property name="text">
      <string>Registration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Last name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="last_name_form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;
  background-color: rgb(172, 172, 172);
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
color:rgb(0, 0, 0);
font-size: 12px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>First name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="first_name_form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;
  background-color: rgb(172, 172, 172);
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
color:rgb(0, 0, 0);
font-size: 12px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Middle name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="middle_name_form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>5</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;
  background-color: rgb(172, 172, 172);
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
color:rgb(0, 0, 0);
font-size: 12px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Phone number</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="phone_number_form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>8</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;
  background-color: rgb(172, 172, 172);
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
color:rgb(0, 0, 0);
font-size: 12px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>8</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;
  background-color: rgb(172, 172, 172);
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
color:rgb(0, 0, 0);
font-size: 12px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Enter your password again</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_again_form">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>8</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;
  background-color: rgb(172, 172, 172);
  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
color:rgb(0, 0, 0);
font-size: 12px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="invalid_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    padding: 0 24px;
    margin: 0 15px 15px 0;
    border: none;
    border-radius: 3px;
    height: 37px;
    font-size: 14px;
    text-transform: uppercase;
    font-weight: normal;
    text-decoration: none;
    color: #fff;
    
	background-color: rgb(39, 53, 83);
}

QPushButton:hover {
    box-shadow: 0px 1px 1px #cfcfcf, 0px 2px 5px #cfcfcf;
}

QPushButton:pressed {
    background-color: rgb(58, 80, 124);
}

QPushButton:disabled {
    background-color: #aed2bc;
    color: #444;
}
</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
